recipe-server/client/control/components/action_fields/HeartbeatFields.js

```diff
@@ -117,6 +121,6 @@ export class HeartbeatFields extends ActionFields {
 
 export default connect(
   state => ({
-    recipeArguments: selector(state, 'arguments'),
+    recipeArguments: selector(state, 'arguments') || {},
```

Comment:
I had gotten errors/warnings when testing the RecipeForm, so I felt compelled to fix 'em. I can pull the changes out into a different PR, though.